modules/gapi/src/backends/ie/giebackend.cpp

```diff
@@ -1018,8 +1041,18 @@ struct InferROI: public cv::detail::KernelTag {
 
                         IE::Blob::Ptr this_blob = extractBlob(*ctx, 1);
 
-                        req.SetBlob(*(ctx->uu.params.input_names.begin()),
-                                IE::make_shared_blob(this_blob, toIE(this_roi)));
+                        // NB: In case importNetwork preprocessing must be
+                        // passed as SetBlob argument.
+                        if (ctx->uu.params.kind == cv::gapi::ie::detail::ParamDesc::Kind::Load) {
+                            req.SetBlob(*(ctx->uu.params.input_names.begin()),
+                                    IE::make_shared_blob(this_blob, toIE(this_roi)));
+                        } else {
+                            IE::PreProcessInfo info;
+                            info.setResizeAlgorithm(IE::RESIZE_BILINEAR);
+                            req.SetBlob(*(ctx->uu.params.input_names.begin()),
+                                    IE::make_shared_blob(this_blob, toIE(this_roi)),
```

Comment:
And here

Reply:
???

Reply:
note relevant since is was moved to separate function as you advised

Reply:
Ok.